kornia/geometry/warp/homography_warper.py

```diff
@@ -264,3 +292,41 @@ def normalize_homography(dst_pix_trans_src_pix: torch.Tensor,
         dst_norm_trans_dst_pix @ (dst_pix_trans_src_pix @ src_pix_trans_src_norm)
     )
     return dst_norm_trans_src_norm
+
+
+def normalize_homography3d(dst_pix_trans_src_pix: torch.Tensor,
```

Comment:
We should move this to another place. This are not 3x3  homographies anymore. Maybe to kornia.geometry.linalg ? @ducha-aiki

Reply:
I think, better to have all homographies in a single place actually

Reply:
now have also `kornia.geometry.homography` as well :)